plugins/proofpoint_tap/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: proofpoint_tap
 title: Proofpoint TAP
 description: Parse Proofpoint Targeted Attack Protection (TAP) alerts
-version: 4.1.1
+version: 4.1.2
 connection_version: 4
 supported_versions: ["Proofpoint TAP API v2", "Tested on 2023-06-22"]
```

Comment:
This might be worth updating to say that it was last tested in January 2024

Reply:
Updated now, good catch.